src/main/java/de/zalando/aruha/nakadi/service/Registry.java

```diff
@@ -0,0 +1,32 @@
+package de.zalando.aruha.nakadi.service;
+
+import com.google.common.collect.ImmutableList;
+import de.zalando.aruha.nakadi.domain.PartitionResolutionStrategy;
+import de.zalando.aruha.nakadi.partitioning.PartitioningStrategy;
+
+import java.util.List;
+
+public class Registry {
```

Comment:
`Registry` sounds so generic and can be basically everything. :) Maybe `StrategiesRegistry` or a dedicated `PartitioningStrategiesRegistry`?